### DISCUSSION TITLE
GPRMC

### DISCUSSION THREAD

**Discussion by PhilipSargent** [semuconsulting/pynmeagps#49] [Ideas and Feature Requests]
Great stuff.

I see that the gpxtracker example doesn't yet convert GPRMC statements into GPX.

I could have a go at becoming a github contributor and adding this -
if someone else is not already doing this sort of thing.

**Comment by semuadmin** [comment]
@PhilipSargent 

Go for it. But bear in mind there's probably no point in updating a GPX track from multiple NMEA position sentences within the same GNSS epoch as they will all contain the same lat/lon and time - so maybe pick _one_ of _either_ GGA, GLL, GNS or RMC.

Also, although not as critical for example code, please try to observe the [CONTRIBUTOR GUIDELINES](https://github.com/semuconsulting/pynmeagps/blob/master/CONTRIBUTING.md) - if you have any queries on these, get back to me.
